tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -392,3 +394,77 @@ def test_harvesting_arxiv_workflow_accepted(
         obj = workflow_object_class.get(obj_id)
         # It was accepted
         assert obj.status == ObjectStatus.COMPLETED
+
+        # Check the conversion for the legacy sys
+        marc_json = hep2marc.do(obj.data)
+        marcxml = legacy_export_as_marc(marc_json)
+        record = create_record(marcxml)
```

Comment:
This is not the best record to test, because in INSPIRE we don't have it, so the things that in the assertions are on the left actually come from the current implementation.

So this is only testing that we don't regress from here, not that we create records that are the same as we have on INSPIRE...